test/functional/framework.go

```diff
@@ -0,0 +1,138 @@
+package functional
+
+import (
+	"fmt"
+	"math/rand"
+	"strings"
+	"time"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/ViaQ/logerr/log"
+	"github.com/openshift/cluster-logging-operator/internal/pkg/generator/forwarder"
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/pkg/certificates"
+	"github.com/openshift/cluster-logging-operator/pkg/constants"
+	"github.com/openshift/cluster-logging-operator/pkg/utils"
+	"github.com/openshift/cluster-logging-operator/test/client"
+	"github.com/openshift/cluster-logging-operator/test/helpers/oc"
+	"github.com/openshift/cluster-logging-operator/test/runtime"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var (
+	maxDuration time.Duration
+)
+
+//FluentddFunctionalFramework deploys stand alone fluentd with the fluent.conf as generated by input ClusterLogForwarder CR
+type FluentdFunctionalFramework struct {
+	Name      string
+	Namespace string
+	Conf      string
+	image     string
+	labels    map[string]string
+	Forwarder *logging.ClusterLogForwarder
+	test      *client.Test
+	pod       *corev1.Pod
+}
+
+func init() {
+	maxDuration, _ = time.ParseDuration("2m")
+}
+
+func NewFluentdFunctionalFramework() *FluentdFunctionalFramework {
+	opt := log.WithVerbosity(uint8(9))
```

Comment:
@jcantrill Just this have to be fixed or can be set by external ENV variable ?

Reply:
this could be evaluatedl like https://github.com/openshift/cluster-logging-operator/pull/751 but we don't have  a single place to set this.  you will only see the logs if the test fails though i would prefer it to not be fixed like this

Reply:
Updated to depend on env var